flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/exception/LocalTransportException.java

```diff
@@ -18,17 +18,20 @@
 
 package org.apache.flink.runtime.io.network.netty.exception;
 
+import org.apache.flink.runtime.clusterframework.types.ResourceID;
+
 import java.net.SocketAddress;
 
 public class LocalTransportException extends TransportException {
 
     private static final long serialVersionUID = 2366708881288640674L;
 
-    public LocalTransportException(String message, SocketAddress address) {
-        super(message, address);
+    public LocalTransportException(String message, SocketAddress address, ResourceID resourceID) {
```

Comment:
Why does a `LocalTransportException` need a resourceId? I think the purpose of printing resource id is to help identify the remote TM that the broken connection connected to. For a local exception, there's no remote TM.

Reply:
make sense, here will be modified.